test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
It would generate a warning, plus killing all the positive cases below (because they are no longer `AST_SymolRef.undeclared()` 😅